### ISSUE TITLE
bytea field causing stacktrace

### ISSUE THREAD

**Issue by apiszcz** [dalibo/pg_activity#302]
Can pg_activity ignore columns?
Thanks.

```
  File "/usr/local/bin/pg_activity", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.8/dist-packages/pgactivity/cli.py", line 299, in main
    ui.main(term, dataobj, host, options, dsn)
  File "/usr/local/lib/python3.8/dist-packages/pgactivity/ui.py", line 196, in main
    pg_procs.set_items(data.pg_get_activities(ui.duration_mode))
  File "/usr/local/lib/python3.8/dist-packages/pgactivity/data.py", line 261, in pg_get_activities
    cur.execute(
  File "/usr/local/lib/python3.8/dist-packages/psycopg2/extras.py", line 146, in execute
    return super().execute(query, vars)
psycopg2.errors.InvalidTextRepresentation: invalid input syntax for type bytea
```

**Comment by blogh** [comment]
Hi which version are you using ?

**Comment by apiszcz** [comment]
Python 3.9.6
pg_activity 2.3.1
PostGres 14.4

--
Testing on Windows 10 client i get this
pg_activity -h localhost -U postgres dbname
FATAL: Platform not supported.

**Comment by apiszcz** [comment]
I have pg_activity running on another system, very nice! Nice work and definitely appreciated.

**Comment by blogh** [comment]
Do you have an example query please ?

**Comment by apiszcz** [comment]
select encode(img::bytea, 'base64') as imgdata, 
from imgtable

**Comment by blogh** [comment]
I can't reproduce the error with either 2.3.1 or master.

Could you [test with the master branch](https://github.com/dalibo/pg_activity#installation-from-the-git-repository) ? we pushed the PR #290, which fixes some of these errors, but it's not released yet.

**Comment by blogh** [comment]
And thanks for the query and report !

**Comment by alepaes1975** [comment]
In one of my system (maybe) I have a similar problem (pg_activity 3.0):

`Traceback (most recent call last):
  File "/usr/local/bin/pg_activity", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.9/dist-packages/pgactivity/cli.py", line 407, in main
    ui.main(term, dataobj, host, args)
  File "/usr/local/lib/python3.9/dist-packages/pgactivity/ui.py", line 205, in main
    pg_procs.set_items(data.pg_get_activities(ui.duration_mode))
  File "/usr/local/lib/python3.9/dist-packages/pgactivity/data.py", line 434, in pg_get_activities
    cur.execute(
  File "/usr/local/lib/python3.9/dist-packages/psycopg2/extras.py", line 146, in execute
    return super().execute(query, vars)
psycopg2.errors.SyntaxError: syntax error at or near "\"
LINE 15:       convert_from(replace(a.query, '\', '\\')::bytea, coale...
                                                   ^
`
This error ocurs when  **standard_conforming_strings =off;**

**Comment by deric** [comment]
I've noticed the same issue on Debian 10, `pg-activity` installed from deb package:
```
Traceback (most recent call last):
  File "/usr/bin/pg_activity", line 11, in <module>
    load_entry_point('pg-activity==2.3.1', 'console_scripts', 'pg_activity')()
  File "/usr/lib/python3/dist-packages/pgactivity/cli.py", line 299, in main
    ui.main(term, dataobj, host, options, dsn)
  File "/usr/lib/python3/dist-packages/pgactivity/ui.py", line 196, in main
    pg_procs.set_items(data.pg_get_activities(ui.duration_mode))
  File "/usr/lib/python3/dist-packages/pgactivity/data.py", line 265, in pg_get_activities
    "dbname_filter": self.filters.dbname,
  File "/usr/lib/python3/dist-packages/psycopg2/extras.py", line 146, in execute
    return super(DictCursor, self).execute(query, vars)
psycopg2.errors.InvalidTextRepresentation: invalid input syntax for type bytea
```
PostgreSQL 13
Python 3.7.3

**Comment by blogh** [comment]
I thanks for reporting

**Comment by blogh** [comment]
@alepaes1975 please try this 

```
postgresql://postgres@localhost/postgres?options=-c%20standard_conforming_strings%3Don
```
see https://www.postgresql.org/docs/14/libpq-connect.html#id-1.7.3.8.3.6 there is an example about options.

**Comment by jonjensen** [comment]
This happened to me too. Today's the first time I've ever run `pg_activity`.

PostgreSQL 12 on CentOS 7, installed via PGDG RPM.

Backtrace:

```
Traceback (most recent call last):
  File "/bin/pg_activity", line 11, in <module>
    load_entry_point('pg-activity==2.3.1', 'console_scripts', 'pg_activity')()
  File "/usr/lib/python3.6/site-packages/pgactivity/cli.py", line 299, in main
    ui.main(term, dataobj, host, options, dsn)
  File "/usr/lib/python3.6/site-packages/pgactivity/ui.py", line 196, in main
    pg_procs.set_items(data.pg_get_activities(ui.duration_mode))
  File "/usr/lib/python3.6/site-packages/pgactivity/data.py", line 265, in pg_get_activities
    "dbname_filter": self.filters.dbname,
  File "/usr/lib64/python3.6/site-packages/psycopg2/extras.py", line 146, in execute
    return super(DictCursor, self).execute(query, vars)
psycopg2.errors.InvalidTextRepresentation: invalid input syntax for type bytea
```

**Comment by blogh** [comment]
Hi @jonjensen !

Thanks for reporting.

Do you have a scenario we could reproduce for this error ? 
We are trying to fix this (#328), but we struggle to have a test case.

**Comment by rafaelorafaelo** [comment]
> pg_activity "postgresql://postgres@localhost/postgres?standard_conforming_strings=on"

This should work
pg_activity "postgresql://postgres@localhost/postgres?options=-c%20standard_conforming_strings%3Don"

**Comment by vitich** [comment]
Got the same...
```
~$ pg_activity -d mydb
Traceback (most recent call last):
  File "/usr/bin/pg_activity", line 11, in <module>
    load_entry_point('pg-activity==2.3.1', 'console_scripts', 'pg_activity')()
  File "/usr/lib/python3/dist-packages/pgactivity/cli.py", line 299, in main
    ui.main(term, dataobj, host, options, dsn)
  File "/usr/lib/python3/dist-packages/pgactivity/ui.py", line 196, in main
    pg_procs.set_items(data.pg_get_activities(ui.duration_mode))
  File "/usr/lib/python3/dist-packages/pgactivity/data.py", line 265, in pg_get_activities
    "dbname_filter": self.filters.dbname,
  File "/usr/lib/python3/dist-packages/psycopg2/extras.py", line 146, in execute
    return super(DictCursor, self).execute(query, vars)
psycopg2.errors.InvalidTextRepresentation: invalid input syntax for type bytea

```

**Comment by dlax** [comment]
> Got the same...
> 
> ```
> ~$ pg_activity -d mydb
> Traceback (most recent call last):
>   File "/usr/bin/pg_activity", line 11, in <module>
>     load_entry_point('pg-activity==2.3.1', 'console_scripts', 'pg_activity')()
> [...]
> ```

This got fixed in version 3.1.0 (and, for the record, the latest release is 3.4.2); you might want to upgrade your installation.
